gcloud-sdk/Cargo.toml

```diff
@@ -20,6 +20,7 @@ default = ["tls-roots"]
 tls-roots = ["tonic/tls-roots", "reqwest/rustls-tls-native-roots"]
 tls-webpki-roots = ["tonic/tls-webpki-roots", "reqwest/rustls-tls-webpki-roots"]
 rest = ["serde_with"]
+external-account-aws = ["aws-config", "aws-sigv4", "aws-credential-types", "percent-encoding"]
```

Comment:
Make external-account auth optional feature because it brings a lot of dependencies, which causes slow build time.